<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TaskFrame</class>
 <widget class="QWidget" name="TaskFrame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>823</width>
    <height>574</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton {
    background-color: #ffffff;
    border: 1px solid #dcdfe6;
    padding: 10px;
    border-radius: 5px;
}

QPushButton:hover {
    background-color: #ecf5ff;
    color: #409eff;
}

QPushButton:pressed, QPushButton:checked {
    border: 1px solid #3a8ee6;
    color: #409eff;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTableWidget" name="tableWidget">
     <property name="autoScroll">
      <bool>true</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <column>
      <property name="text">
       <string comment="方法" extracomment="飞飞飞">销售单号</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>客户名称</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>客户订单号</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>订单金额</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>业务员</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>创建时间</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_2" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QToolButton" name="toolButton_7">
        <property name="minimumSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="toolTip">
         <string>首页</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normalon>:/logo/pic/first.png</normalon>
         </iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="toolButton_8">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>24</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="toolTip">
         <string>上一页</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normalon>:/logo/pic/pre.png</normalon>
         </iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="toolButton_9">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>24</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="toolTip">
         <string>下一页</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normalon>:/logo/pic/next.png</normalon>
         </iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="toolButton_10">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>24</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="toolTip">
         <string>末页</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normalon>:/logo/pic/end.png</normalon>
         </iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line_5">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="toolButton_11">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>24</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="toolTip">
         <string>跳转到</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normalon>:/logo/pic/jump.png</normalon>
         </iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="pageEdit">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>24</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>24</height>
         </size>
        </property>
        <property name="maximum">
         <number>9999</number>
        </property>
        <property name="stepType">
         <enum>QAbstractSpinBox::AdaptiveDecimalStepType</enum>
        </property>
        <property name="displayIntegerBase">
         <number>10</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line_6">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>--</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line_7">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_12">
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>32</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="toolTip">
         <string>请选择一条记录并开始扫描</string>
        </property>
        <property name="text">
         <string>开始扫描</string>
        </property>
        <property name="icon">
         <iconset>
          <normalon>:/logo/pic/scanIco.png</normalon>
         </iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line_8">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QScrollArea" name="scrollArea_2">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>200</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>200</height>
      </size>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>782</width>
        <height>202</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <layout class="QFormLayout" name="formLayout_3">
         <property name="fieldGrowthPolicy">
          <enum>QFormLayout::ExpandingFieldsGrow</enum>
         </property>
         <property name="rowWrapPolicy">
          <enum>QFormLayout::WrapLongRows</enum>
         </property>
         <property name="labelAlignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
         <property name="horizontalSpacing">
          <number>3</number>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="Label_3">
           <property name="text">
            <string>销售单号</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="LineEdit_7">
           <property name="text">
            <string/>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="Label2_2">
           <property name="text">
            <string>客户名称</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="Label3_2">
           <property name="text">
            <string>客户订单号</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="LineEdit_8">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="Label54_2">
           <property name="text">
            <string>订单金额</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLineEdit" name="LineEdit_9">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="Label8_2">
           <property name="text">
            <string>业务员</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QLineEdit" name="LineEdit_10">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="LineEdit_11">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="Label_4">
           <property name="text">
            <string>价税</string>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QLineEdit" name="LineEdit_12"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../images.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>tableWidget</sender>
   <signal>itemSelectionChanged()</signal>
   <receiver>TaskFrame</receiver>
   <slot>dataRowSelected()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>154</x>
     <y>172</y>
    </hint>
    <hint type="destinationlabel">
     <x>282</x>
     <y>721</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tableWidget</sender>
   <signal>itemDoubleClicked(QTableWidgetItem*)</signal>
   <receiver>TaskFrame</receiver>
   <slot>openTaskForm()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>339</x>
     <y>226</y>
    </hint>
    <hint type="destinationlabel">
     <x>326</x>
     <y>708</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_12</sender>
   <signal>clicked()</signal>
   <receiver>TaskFrame</receiver>
   <slot>openTaskForm()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>733</x>
     <y>342</y>
    </hint>
    <hint type="destinationlabel">
     <x>865</x>
     <y>78</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toolButton_7</sender>
   <signal>clicked()</signal>
   <receiver>TaskFrame</receiver>
   <slot>firstPage()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>417</x>
     <y>331</y>
    </hint>
    <hint type="destinationlabel">
     <x>834</x>
     <y>284</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toolButton_8</sender>
   <signal>clicked()</signal>
   <receiver>TaskFrame</receiver>
   <slot>prePage()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>446</x>
     <y>337</y>
    </hint>
    <hint type="destinationlabel">
     <x>842</x>
     <y>252</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toolButton_9</sender>
   <signal>clicked()</signal>
   <receiver>TaskFrame</receiver>
   <slot>nextPage()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>484</x>
     <y>336</y>
    </hint>
    <hint type="destinationlabel">
     <x>840</x>
     <y>217</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toolButton_10</sender>
   <signal>clicked()</signal>
   <receiver>TaskFrame</receiver>
   <slot>endPage()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>511</x>
     <y>335</y>
    </hint>
    <hint type="destinationlabel">
     <x>838</x>
     <y>189</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toolButton_11</sender>
   <signal>clicked()</signal>
   <receiver>TaskFrame</receiver>
   <slot>jumpPage()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>561</x>
     <y>334</y>
    </hint>
    <hint type="destinationlabel">
     <x>835</x>
     <y>159</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>firstPage()</slot>
  <slot>prePage()</slot>
  <slot>nextPage()</slot>
  <slot>endPage()</slot>
  <slot>jumpPage()</slot>
  <slot>openTaskForm()</slot>
  <slot>dataRowSelected()</slot>
 </slots>
</ui>
